src/actions/tests/__snapshots__/ast.js.snap

```diff
@@ -1,6 +1,96 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ast getOutOfScopeLocations with selected line 1`] = `
+Object {
```

Comment:
hmm - not sure if we need this! it's a flaky test.

Take it out and we should be able to land it